content/go/concepts/goroutines/goroutines.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: 'Goroutines'
+Description: 'Goroutines are functions and methods that run concurrently with other functions and methods.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Threads'
+  - 'Concurrency'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Goroutines** are functions and methods that run concurrently with other functions and methods.
+
+## The difference between threads and goroutines
```

Comment:
I'd get rid of the header and incorporate the following paragraph into the introduction.